docs/install/lotus/README.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Install Lotus
+description: Set up the Lotus Filecoin client on your computer, ready to be used as a single node or in a mining operation.
+---
+
+# Install Lotus
+
+Lotus is an implementation of the Filecoin specification written in Go. You can run the Lotus client to interact with the Filecoin network. The Lotus project is created and maintained by Protocol Labs, the creators of IPFS, libp2p, and Filecoin. This section covers how to install the Lotus client on your machine.
+
+The Lotus client can be used for one of three use-cases:
+
+| Client        | Purpose                                                                                                                                                                         |
+| ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Lotus node    | An interface between your computer and the Filecoin network. All transactions, queries, and interactions with the Filecoin network must pass through a Lotus node.              |
+| Storage miner | The Lotus storage miner queries a Lotus node for the latest tipset on-chain, produces a block on that tipset, and submits it to the node to be broadcast to the entire network. |
+| Seal worker   | An extra process that can offload heavy processing tasks from your Lotus Storage miner.                                                                                         |
+
+All three of these use-cases use the same Lotus client, but they differ in how they are configured _after_ the client has been installed. To make the install go as smoothly as possible, read through the [software and hardware requirements](./requirements) first.
+
+## Linux
+
+1.  Install the dependencies:
```

Comment:
I feel like it's missing instructions to install `go`, it's also a required dependency.

In lotus docs notes I added:

```
Setup
- Add a reminder for users to install Go and add it to $PATH
- Add a remark that users should not install Go 1.15
```

Reply:
Requirements are on the [Requirements page](https://deploy-preview-305--filecoin-docs.netlify.app/install/lotus/requirements/#hardware). I'm going to resolve this conversation for now because we _really just need to get this merged_. Feel free to create an issue to discuss next steps regarding Requirements and installation steps.